bazel/linux/defs.bzl

```diff
@@ -311,3 +311,28 @@ Example:
         ),
     },
 )
+
+def kernel_test(name, repo_name, module, kernel_image, rootfs_image):
```

Comment:
Do we have a README indicating how to use enkit from another repo? If we had one, it'd probably say to import it as @enkit. Can you default repo_name to be = to `@enkit`? So it works for those that use the `default` name.

Reply:
Yes, I agree that @enkit is a sane default.